src/pages/HomeButton.jsx

```diff
@@ -0,0 +1,7 @@
+
+
+export const HomeButton = () => {
+  return (
+    <div></div>
+  )
```

Comment:
I see you added the text to the button in the react router dom "Link" component - how come you didn't add it here? For a cleaner an easier to read code you could also have added a text-prop instead, and add the text that way :). Perhaps you could also make the button an actual button-element instead?